### PR TITLE
Added Linker Test for Jump Table Generation

### PR DESCRIPTION
**Pull request by NandniJamnadas**
Signed-off-by: NandniJamnadas <nandni.jamnadas@embecosm.com>

### CODE REVIEW

File: ld/testsuite/ChangeLog.COREV [hunk 1]
```diff
@@ -1,3 +1,10 @@
+2022-08-23  Nandni Jamnadas  <nandni.jamnadas@embecosm.com>
+
+	* ld/testsuite/ld-riscv-elf/jump-table.s: Added tests 
+	to check that the jump tables symbols are generated in 
+	.rodata rather than .text.
+
```
Comment (pietraferreira) [inline ld/testsuite/ChangeLog.COREV:6]: Extra line here

**Comment by pietraferreira** [comment]
LGTM, I'd just rename the test file to something more specific like `zc-jump-table-gen` or `zcmt-jt-gen`

File: ld/testsuite/ld-riscv-elf/jump-table.s [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    call  callee
+    call  callee
+    call  callee
+    call  callee
+    call  callee
+    call  callee
+    call  callee
+    call  callee
+    .size _start, .-_start
+
```
Comment (CharKeaney) [inline ld/testsuite/ld-riscv-elf/jump-table.s]: No line at end of file

File: ld/testsuite/ChangeLog.COREV (hunk 1 above)
Comment (CharKeaney) [reply ld/testsuite/ChangeLog.COREV:6]: Fixed now so LGTM.

**Review by CharKeaney — COMMENTED** [review]
LGTM.

**Pull request merged**